lib/Agrammon/Web/Routes.pm6

```diff
@@ -268,14 +246,33 @@ sub api-routes (Str $schema, $ws) {
                 }
             }
         }
+        operation 'getExcelExport', -> LoggedIn $user {
+            request-body -> %params {
+                response.append-header(
+                        'Content-disposition',
+                        "attachment; filename=%params<datasetName>.xlsx"
```

Comment:
Is there any certainty on what `datasetName` could be? If it may contain newlines, for example, then we're vulnerable to header injection attacks here. Perhaps safest is to have an allowlist or chars and filter them out, e.g. 

```
"attachment; filename=%params<datasetName>.subst(/<-[\w_-.]>/, '', :g).xlsx"
```

Reply:
Good point. Fixed.